pipeline/matcher_merger/merger/src/test/scala/weco/pipeline/merger/services/MergerWorkerServiceTest.scala

```diff
@@ -136,7 +137,7 @@ class MergerWorkerServiceTest
     }
   }
 
-  it("always sends the highest version of a Work") {
+  it("send versions of a Work even if they are not the latest") {
```

Comment:
I think this is a pre-existing issue with this test, but it took me a few reads-through to realise that although `olderWork` and `newerWork` are obviously versions of the same thing, `work` is something else entirely, and that was why the result now has two identifiers in it, when previously one of them was discarded.

Does this make sense?

```
it("sends a Work even if the matched version is not the latest") {
    withMergerWorkerServiceFixtures {
      case (retriever, QueuePair(queue, dlq), senders, _, index) =>
        val workA = identifiedWork()
        val workB = identifiedWork()
        val newerWorkB =
          identifiedWork(canonicalId = workB.state.canonicalId)
            .withVersion(workB.version + 1)
        val matcherResult = createMatcherResultWith(Set(Set(workA, workB)))

        retriever.index ++= Map(workA.id -> workA, workB.id -> newerWorkB)

        sendNotificationToSQS(
          queue = queue,
          message = matcherResult
        )

        eventually {
          assertQueueEmpty(queue)
          assertQueueEmpty(dlq)
          getWorksSent(senders) should contain allOf (workA.id, workB.id)
          index shouldBe Map(
            workA.id -> Left(
              workA.transition[Merged](matcherResult.createdTime)
            ),
            workB.id -> Left(
              newerWorkB.transition[Merged](matcherResult.createdTime)
            )
          )
        }
    }
  }
```

Reply:
Yes, this one also puzzled me for a bit so I think worth updating the test as described.

Reply:
This is the one I found confusing! To check my understanding, what's being tested here is that if we have two works that should merged, we send them to the queue even if their versions are different. Previously workA would have been held up because it has an older version?

Reply:
> we send them to the queue even if their versions are different. Previously workA would have been held up because it has an older version?

Yes, I think so. It might be clearer to say "we send them to the queue even if the version are different when we compare the version of a work we get sent by the matcher and what we find in the identified index for that work".